src/transport/smtp/mod.rs

```diff
@@ -284,8 +284,6 @@ impl SmtpTransport {
     /// * No authentication
     /// * A 60 seconds timeout for smtp commands
     /// * Port 587
-    ///
-    /// Consider using [`SmtpTransport::new`] instead, if possible.
```

Comment:
I think this is a [leftover from lettre 0.9](https://docs.rs/lettre/0.9.3/lettre/smtp/struct.SmtpTransport.html#method.new)!?

Reply:
Nevermind, wasn't paying attention